assets/src/components/amp-support/plugins.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { ListItems } from '../list-items';
+
+/**
+ * To render plugins information on site support page.
+ *
+ * @param {Object} props      Component props.
+ * @param {Object} props.data Plugins data.
+ * @return {JSX.Element|null} HTML markup for plugins data.
+ */
+export function Plugins( { data: plugins } ) {
+	if ( 'object' !== typeof plugins ) {
+		return null;
+	}
+
+	plugins = Object.values( plugins );
+
+	return (
+		<details open={ false }>
+			<summary>
+				{ __( 'Plugins', 'amp' ) }
+				{ ` (${ plugins.length || 0 })` }
```

Comment:
This should use `sprintf` and no logical OR operator as described in other comments.